Complete the community governance (#157)
kubeedge/community: GOVERNANCE.md

Special Teams also do not have open membership and do not always operate in the open. The technical steering committee can form special team as needed, for
bounded or unbounded duration. Membership of a special team is decided by the technical steering committee,
however, all teams members must be [community members]. A special team must has a charter, and need to report to the
technical steering committee periodically.

### Special Interest Groups (SIGs)

The KubeEdge project is organized primarily into Special Interest Groups, or
SIGs. Each SIG is comprised of individuals from multiple companies and
organizations, with a common purpose of advancing the project with respect to a
specific topic. SIGs are **permanent** until dissolved by TSC.

The goal is to enable a distributed decision structure and code ownership,
as well as providing focused forums for getting work done, making decisions,
and on-boarding new Contributors. Every identifiable part of the project
(e.g. repository, subdirectory, API, test, issue, PR) is intended to be
owned by some SIG.

SIGs must have open membership and always operate **in the open**.The leadership(SIG chairs and technical leads) 

> changes of SIGs is decided by the technical steering committee, for bounded or unbounded duration. And all 

Reviewer: ```suggestion
changes of SIGs need approval by the technical steering committee, for bounded or unbounded duration. And all 
```

the operation of the SIG and for communication and coordination with the other
SIGs, and the broader community.

> 

Reviewer 2: unnecessary change?

Author: Done :) PTAL

- Use the above forums as the primary means of working, communicating, and collaborating, as opposed
  to private emails and meetings.

#### Technical Leads

Technical Leads are responsible for leading the SIG in correspondence with its technical 
alignment. This alignment includes both internal to the SIG and, more significantly, external
to the entire project.

SIGs can decide on their own if they want to add Technical Leads to their charter or not. 

> Depending on the overall size of the SIG, around two to three people can be chosen by the 

Reviewer: ```suggestion
Depending on the overall size of the SIG, around two to three people can be nominated by the 
```

SIG Chairs to support the technical aspects of the group. To be able to fulfill their role, 
a Technical Lead should have the same set of permissions as a Chair.

#### Sub Projects

Specific work efforts within SIGs are divided into **subprojects**.
Every part of the code and documentation must be owned by
some subproject. Some SIGs may have a single subproject, but many SIGs
have multiple significant subprojects with distinct (though sometimes
overlapping) sets of reviewers and approvers, who act as
subproject’s technical leaders: responsible for vision and direction
and overall design, choose/approve project approvers/reviewers,
field technical escalations, etc.

### Working Groups (WGs)

Working Groups provide a formal avenue for disparate groups to collaborate around a common problem, 

> craft a balanced position, and **dissolved** after the problem solved by TSC. Because they represent the

Reviewer: ```suggestion
craft a balanced position, and **dissolved** by TSC after the problem solved. Because they represent the
```

Author: Done.
